examples/authentication/README.md

```diff
@@ -0,0 +1,105 @@
+# Authentication
+
+This example demonstrates how to implement a GraphQL server with an email-password-based authentication workflow based on Prisma & [graphql-yoga](https://github.com/graphcool/graphql-yoga).
+
+## Get started
+
+> **Note**: `prisma` is listed as a _development dependency_ and _script_ in this project's [`package.json`](./package.json). This means you can invoke the Prisma CLI without having it globally installed on your machine (by prefixing it with `yarn`), e.g. `yarn prisma deploy` or `yarn prisma playground`. If you have the Prisma CLI installed globally (which you can do with `npm install -g prisma`), you can omit the `yarn` prefix.
+
+### 1. Download the example & install dependencies
+
+Clone the Prisma monorepo and navigate to this directory or download _only_ this example with the following command:
+
+```sh
+curl https://codeload.github.com/graphcool/prisma/tar.gz/application-server | tar -xz --strip=2 prisma-master/examples/application-server
+```
+
+Next, navigate into the downloaded folder and install the NPM dependencies:
+
+```sh
+cd application-server
+yarn install
+```
+
+### 2. Deploy the Prisma database service
+
+You can now [deploy](https://www.prisma.io/docs/reference/cli-command-reference/database-service/prisma-deploy-kee1iedaov) the Prisma service (note that this requires you to have [Docker](https://www.docker.com) installed on your machine - if that's not the case, follow the collapsed instructions below the code block):
+
+```sh
+yarn prisma deploy
+```
+
+<details>
+ <summary><strong>I don't have <a href="https://www.docker.com">Docker</a> installed on my machine</strong></summary>
+
+To deploy your service to a demo server (rather than locally with Docker), you need to perform the following steps:
+
+1.  Remove the `cluster` property from `prisma.yml`
```

Comment:
this section is outdated

Reply:
Should I provide instructions inline or link the quick-start https://www.prisma.io/docs/quickstart/ link here?